packages/cli/test/dev/integration-3.test.ts

```diff
@@ -104,13 +104,38 @@ test(
 
 test('[vercel dev] 08-hugo', async () => {
   if (process.platform === 'darwin') {
-    // Update PATH to find the Hugo executable installed via GH Actions
+    // 1. run the test without Hugo in the PATH
+    let tester = await testFixtureStdio(
+      '08-hugo',
+      async () => {
+        throw new Error('Expected dev server to fail to be ready');
+      },
+      {
+        readyTimeout: 2000,
+
+        // Important: for the first test, we MUST deploy this app so that the
+        // framework (e.g. Hugo) will be detected by the server and associated
+        // with the project since `vc dev` doesn't do framework detection
```

Comment:
Put a `vercel.json` file with `"framework": "hugo"` in the fixture. `vc dev` supports project configuration overrides now.